packages/transaction-controller/src/utils/gas.ts

```diff
@@ -136,8 +132,12 @@ async function getGas(
     request.ethQuery,
   );
 
-  if (isCustomNetwork) {
-    log('Using original estimate as custom network');
+  if (isCustomNetwork || simulationFails) {
+    log(
+      isCustomNetwork
+        ? 'Using original estimate as custom network'
+        : 'Using original estimate as simulation failed',
```

Comment:
Should this read `fallback estimate` for clarity?

Reply:
Done